### PR TITLE
doc/add prometheus grafana integration doc

### PR DESCRIPTION
**Pull request by kevincai**

### CODE REVIEW

File: doc/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Table Of Content
+
+## Index
+
++ [API Reference](./api.md)
++ HOWTO
+  + [Change Root Password HOWTO](./change_root_password_howto.md)
```
Comment (yandongxiao) [inline doc/README.md:7]: use dashed line?

File: doc/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  + [Init Root Password When First Deploy HOWTO](./initialize_root_password_howto.md)
```
Comment (yandongxiao) [inline doc/README.md:8]: ditto

File: doc/integration/integration-prometheus-grafana.md [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+* How to turn on prometheus metrics scrape for the StarRocks cluster
+* How to import StarRocks Grafana dashboard
+
+
+## 1. Prerequisites
+
++ Kubernetes: v1.18+ - v1.26
++ Prometheus helm chart: latest available version
++ Grafana helm chart: latest available version
++ StarRocks operator and helm chart: v1.7.1+
```
Comment (yandongxiao) [inline doc/integration/integration-prometheus-grafana.md:13]: please use helm chart v1.8.0+, becase the values.yaml is not compatible with previous versions

File: doc/integration/integration-prometheus-grafana.md [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+From the navigation path, `"Administration" -> "Configuration" -> "Add Data Source"`, set the HTTP.URL to `http://prometheus-server.default`. Change `default` to the correct namespace if the prometheus service is not deployed in the default namespace. Refer to Grafana doc [here](https://grafana.com/docs/grafana/latest/datasources/prometheus/configure-prometheus-data-source/) for detailed instructions.
+
+## 5. Deploy StarRocks Cluster
+
+Follow the instructions from [StarRocks Helm Chart](https://artifacthub.io/packages/helm/kube-starrocks/kube-starrocks) with some customized values.
+
+Following is an example of the content of the `sr-values.yaml`.
+
+```yaml
+starrocksFESpec:
```
Comment (yandongxiao) [inline doc/integration/integration-prometheus-grafana.md:84]: should be indented and placed below `starrocks`

File: doc/README.md (hunk 2 above)
Comment (kevincai) [reply doc/README.md:8]: done

File: doc/README.md (hunk 1 above)
Comment (kevincai) [reply doc/README.md:7]: done

File: doc/integration/integration-prometheus-grafana.md (hunk 4 above)
Comment (kevincai) [reply doc/integration/integration-prometheus-grafana.md:84]: good catch

File: doc/integration/integration-prometheus-grafana.md (hunk 3 above)
Comment (kevincai) [reply doc/integration/integration-prometheus-grafana.md:13]: 1.8.0 is not released yet :)

File: doc/integration/integration-prometheus-grafana.md (hunk 4 above)
Comment (kevincai) [reply doc/integration/integration-prometheus-grafana.md:85]: add examples for both 1.7.1 and 1.8.0

**Review by yandongxiao — APPROVED** [review]

**Pull request merged**